src/main/java/lotto/model/LottoTicketGenerator.java

```diff
@@ -0,0 +1,18 @@
+package lotto.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoTicketGenerator {
+
+    private static final List<LottoNumber> LOTTO_NUMBERS = IntStream.rangeClosed(LottoNumber.LOWER_BOUND, LottoNumber.UPPER_BOUND)
+            .mapToObj(LottoNumber::new)
+            .collect(Collectors.toList());
```

Comment:
아래와 같이 `static` 블록을 활용하면 조금 더 가독성을 높일 수 있을 것 같아요. 🙂

```suggestion
    private static final List<LottoNumber> LOTTO_NUMBERS;

    static {
        LOTTO_NUMBERS = IntStream.rangeClosed(LottoNumber.LOWER_BOUND, LottoNumber.UPPER_BOUND)
            .mapToObj(LottoNumber::new)
            .collect(Collectors.toList());
    }
```

Reply:
반영하였습니다!